docs/components/modeler/web-modeler/create-a-process-application.md

```diff
@@ -0,0 +1,63 @@
+---
+id: create-a-process-application
+title: Create a process application
+description: Create a process application.
```

Comment:
I think this character length for the description is a bit too small -- can we add a description that's closer to 150?

Reply:
Thanks - done 👍